dev/integration_tests/display_cutout_rotation/test_driver/display_cutout_test_test.dart

```diff
@@ -10,22 +10,29 @@ import 'dart:io';
 
 import 'package:flutter_driver/flutter_driver.dart';
 
-// display_cutout needs a custom driver becuase cutout manipulations needs to be
+// display_cutout needs a custom driver because cutout manipulations needs to be
 // done to a device/emulator in order for the tests to pass.
 Future<void> main() async {
   if (!(Platform.isLinux || Platform.isMacOS)) {
     // Not a fundemental limitation, developer shortcut.
     print('This test must be run on a POSIX host. Skipping...');
     return;
   }
-  final bool adbExists = Process.runSync('which', <String>['adb']).exitCode == 0;
-  if (!adbExists) {
-    print(r'This test needs ADB to exist on the $PATH.');
-    exitCode = 1;
-    return;
+  String adbExecutable = 'adb';
+  final bool adbExistsOnPath = Process.runSync('which', <String>[adbExecutable]).exitCode == 0;
+  if (!adbExistsOnPath) {
+    print(r'ADB does not exist on the $PATH. Falling back to $ANDROID_HOME');
+    adbExecutable = '${Platform.environment['ANDROID_HOME']}/platform-tools/adb';
+    final ProcessResult result = Process.runSync('which', <String>[adbExecutable]);
+    final bool adbExistsInAndroidSdk = result.exitCode == 0;
+    if (!adbExistsInAndroidSdk) {
+      print(r'This test needs ADB to exist on the $PATH or in $ANDROID_HOME');
+      exitCode = 1;
+      return;
+    }
   }
   // Test requires developer settings added in 28 and behavior added in 30
-  final ProcessResult checkApiLevel = Process.runSync('adb', <String>[
+  final ProcessResult checkApiLevel = Process.runSync(adbExecutable, <String>[
```

Comment:
These instead should use `shellExec`, which does all the PATH checking you're looking for and finds the `adb` command.  Like this:
https://github.com/flutter/flutter/blob/41c3008afb59d6179fc482e64b74c667096d15b1/dev/devicelab/bin/tasks/fast_scroll_large_images__memory.dart#L30

Reply:
In fact I _think_ this one can use the current Android device (assuming its pointed at an emulator and not the tethered device) `apiLevel`
https://github.com/flutter/flutter/blob/41c3008afb59d6179fc482e64b74c667096d15b1/dev/devicelab/lib/framework/devices.dart#L580

Reply:
Talked out of band and it appears the device lab classes are for use only in that package.